include/hoconsc.hrl

```diff
@@ -34,4 +35,8 @@
 %% Map keys are always strings, `Name' is only for documentation
 -define(MAP(Name, Type), {map, Name, Type}).
 
+%% To avoid not import those function. we provide a macro to call them.
+-define(MK(Type), hoconsc:mk(Type)).
+-define(MK(Type, Meta), hoconsc:mk(Type, Meta)).
```

Comment:
Should we reload all of modules relies on this header file, if it changed and release upgrade is needed?

Reply:
I think we don't need to change this macro anymore, we only change the function implementation `hoconsc:mk/2`. This macro is just a shortcut to a function, too simple to be modified in the future.
The only case I can think of where it might be modified is when `hoconsc:mk/2` function is renamed. But we should add new function to do that instead of changing the name.